.github/workflows/easycloud-build.yml

```diff
@@ -9,6 +9,9 @@ on:
     paths:
       - '.github/workflows/**'
       - 'src/**'
+      - 'IONOS'
```

Comment:
Does it have to be `IONOS/**` like the other directories?

Reply:
it worked without it.. https://github.com/IONOS-Productivity/nc-server/actions/runs/9399967096/job/25888516061?pr=13#step:4:59
..I've added it though